Libplanet/Action/IActionContext.cs

```diff
@@ -70,6 +70,12 @@ public interface IActionContext
         /// </summary>
         HashDigest<SHA256>? PreviousStateRootHash { get; }
 
+        /// <summary>
+        /// Boolean flag indicating if this action was executed as a BlockAction.
```

Comment:
It's trivial though…

```suggestion
        /// Whether this action is executed as a block action.
        /// <c>false</c> if it belongs to a transaction.
```

Reply:
amended https://github.com/planetarium/libplanet/pull/1143/commits/99d1396f3d958304550794d358eff604327ec3e6

Reply:
@riemannulus It seems you've missed the second statement.

Reply:
you're right, I amended 105b7ff one more time